lib/commons/text/accessible-text.js

```diff
@@ -137,34 +138,34 @@ function nonEmptyText(t) {
  * @return {string}
  */
 text.accessibleText = function(element, inLabelledByContext) {
-	//todo: implement shadowDOM
-	var accessibleNameComputation;
-	var encounteredNodes = [];
+	let accessibleNameComputation;
+	const encounteredNodes = [];
+	if (element instanceof Node) {
```

Comment:
I am going to accept this pull request and have opened a separate issue to deal with this https://github.com/dequelabs/axe-core/issues/435